source/initZones/locatorzonesV2.sqf

```diff
@@ -34,15 +34,17 @@ for [{_i=1}, {_i<=zones_number}, {_i=_i+1}] do  // BEGIN "FOR" LOOP --
 
         if (clicked) then {
 
-            _missionPos = [ClickedPos, 0,50,15,0,0.1,0,[],[[0,0],[0,0]]] call BIS_fnc_findSafePos;
+            _posDeMeilleurTruc = [];
+            _posDeMeilleurTruc = [ClickedPos, 0,50,15,0,0.1,0,[],[[0,0],[0,0]]] call BIS_fnc_findSafePos;
```

Comment:
If you don't initialize it that first time, the game won't know it's supposed to be an array, it'll simplify it to a single object. Once it realizes you want an array, it'll give an error and won't work.